PendingReleaseNotes.md

```diff
@@ -12,6 +12,9 @@ v1.7...
 - Add user data protection when deleting Rook-Ceph Custom Resources
   - A CephCluster will not be deleted if there are any other Rook-Ceph Custom resources referencing
     it with the assumption that they are using the underlying Ceph cluster.
+  - A CephObjectStore will not be deleted if there is a bucket present. In addition to protection
```

Comment:
Not protected if there are users. Only buckets. I thought that is what we discussed, but we can revisit the topic.

Reply:
If you are referencing about cephobjectstoreUser/OBC level, IMHO it better to block both for cephobjectstore deletion. But if it only about the backend bucket, then only bucket is need to considered

Reply:
Sorry after reading (resource dependencies)[https://github.com/rook/rook/blob/master/design/ceph/resource-dependencies.md], I understand it is referring to the actual backend bucket.
{bit of out scope of this PR]In the design doc, it has mentioned storageclass dependency. There storageclass created for OBC, do we need to consider that dependency as well?

Reply:
We also opted to ignore storageclasses as dependencies for block/file storage, so I think the same logic applies here.

Reply:
> Not protected if there are users. Only buckets. I thought that is what we discussed, but we can revisit the topic.

Agreed that users wouldn't be protected, I just wasn't clear from this sentence since it mentions "when users have data in the store." I guess it's just referring to their buckets.